boards/arm/stm32/common/src/stm32_nunchuck.c

```diff
@@ -98,10 +98,10 @@ int board_nunchuck_initialize(int devno, int busno)
 
   /* Register the joystick device as /dev/nunchuck0 */
 
-  iinfo("Initialize joystick driver: %s\n", devname);
+  iinfo("Initialize joystick driver: %s\n", devpath);
```

Comment:
`devpath` is empty at this point.  This line should be swapped with 103.

Reply:
Ok